<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QFrame" name="buttonFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <property name="horizontalSpacing">
            <number>6</number>
           </property>
           <property name="verticalSpacing">
            <number>0</number>
           </property>
           <item row="3" column="1">
            <widget class="MDComboBox" name="comboBox_3">
             <property name="labelText" stdset="0">
              <string>Due</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="MDComboBox" name="comboBox_4">
             <property name="labelText" stdset="0">
              <string>Tre</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="MDTextline" name="lineEdit_2"/>
           </item>
           <item row="6" column="0">
            <widget class="MDTextline" name="lineEdit"/>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="buttonSettings">
             <property name="text">
              <string>SETTINGS</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="buttonPrint">
             <property name="text">
              <string>PRINT</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="buttonProfile">
             <property name="text">
              <string>PROFILE</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="buttonInfo">
             <property name="text">
              <string>INFO</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QPushButton" name="buttonRefreshStyle">
             <property name="text">
              <string>REFRESH STYLE</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>Ciao</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Due</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Tre</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Quattro</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QRadioButton" name="radioButton_2">
             <property name="text">
              <string>RadioButton</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QRadioButton" name="radioButton">
             <property name="text">
              <string>Prova</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QTreeWidget" name="treeWidget">
             <column>
              <property name="text">
               <string>1</string>
              </property>
             </column>
             <item>
              <property name="text">
               <string>Padre</string>
              </property>
              <item>
               <property name="text">
                <string>Figlio 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Figlio 2</string>
               </property>
               <item>
                <property name="text">
                 <string>Nipote2a</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Nipote2b</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Nipote2c</string>
                </property>
               </item>
              </item>
              <item>
               <property name="text">
                <string>Figlio 3</string>
               </property>
              </item>
             </item>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>CheckBox</string>
             </property>
            </widget>
           </item>
           <item row="13" column="1">
            <widget class="QCheckBox" name="checkBox_3">
             <property name="text">
              <string>CheckBox</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="MDSwitchLine" name="checkBox_6">
             <property name="text">
              <string>Single-line Item</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="buttonManual">
             <property name="text">
              <string>MANUAL</string>
             </property>
            </widget>
           </item>
           <item row="14" column="0">
            <widget class="QRadioButton" name="radioButton_4">
             <property name="text">
              <string>RadioButton</string>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QCheckBox" name="checkBox_2">
             <property name="text">
              <string>CheckBox</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="MDSwitchLine" name="checkBox_5">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Single-line Item</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="MDListWidget" name="listWidget"/>
           </item>
           <item row="13" column="0">
            <widget class="QRadioButton" name="radioButton_3">
             <property name="text">
              <string>RadioButton</string>
             </property>
            </widget>
           </item>
           <item row="14" column="1">
            <widget class="QCheckBox" name="checkBox_4">
             <property name="text">
              <string>CheckBox</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="MDComboBox" name="comboBox_2">
             <property name="labelText" stdset="0">
              <string>Uno</string>
             </property>
             <item>
              <property name="text">
               <string>Uno</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Single-line Item</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Tre</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="7" column="2">
            <widget class="QSlider" name="verticalSlider">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="9" column="2">
            <widget class="QSlider" name="horizontalSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="8" column="2">
            <widget class="QSlider" name="horizontalSlider_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QSlider" name="verticalSlider_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MDTextline</class>
   <extends>QLineEdit</extends>
   <header>gui/custom-widgets/MDTextline.hpp</header>
  </customwidget>
  <customwidget>
   <class>MDSwitchLine</class>
   <extends>QCheckBox</extends>
   <header>gui/custom-widgets/MDSwitchLine.hpp</header>
  </customwidget>
  <customwidget>
   <class>MDListWidget</class>
   <extends>QListWidget</extends>
   <header>gui/custom-widgets/MDListWidget.hpp</header>
  </customwidget>
  <customwidget>
   <class>MDComboBox</class>
   <extends>QComboBox</extends>
   <header>gui/custom-widgets/MDComboBox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
